src/WebExploitation/SQLInjections/Intro to SQLi.md

```diff
@@ -0,0 +1,94 @@
+# SQL Injections (aka SQLi)
+
+## What is SQLi?
+
+SQL Injection (SQLi) is a security vulnerability that allows attackers to inject malicious SQL code into queries sent to a database. This can lead to unauthorized access to data, data manipulation, or even full control over the database. SQL is a standard language for managing data held in relational database management systems (RDBMS), including MySQL, Oracle, SQL Server, and others.
+
+<img src="../../../assets/SQL-injection-attack-example.png" style="justify-content:center; align-item:center">
+
+## Index
+1. Basis to tackle SQLi (Below)
+2. [Authentication By-Pass](./authentication_bypass.md)
+3. [Union Based Attacks](./union_based_attacks.md)
+4. [Blind Injections](./blind_injection.md)
+5. [Time-Based Blind Injections](./time_based_blind_injections.md)
+6. [Error Based Injections](./error_based_injections.md)
+7. [Second Order Injections](./second_order_injections.md) 
+  
+
+## Basis to Tackle SQLi Challenges
+- The `--` comments out the rest of the query, effectively bypassing the password check and logging the attacker in as any user.
+- Basic SQL syntax
+  1. `SELECT`
+  Extracts data from the given table.
+
+  1. `ORDER BY`
+  Orders the results gotten from SELECT in a specific manner. For
+  example, if one has a table of countries and their populations. One can
+  select the countries starting with the letter R and then order them by
+  their population.
+
+  1. `UNION` clause allows you to group multiple queries together.
+
+  2. `JOIN`
+  Joins data from two tables depending on a certain characteristic on the
+  table. So if a theres two table one with customer IDs and their addresses
+  and another table with customer IDs and their purchases. You can join
+  both tables so that the customer addresses match their purchases.
+  1. `DELETE, INSERT`
+  Allows you to delete data or add new data to a table.
+  1. `AND,OR`
+  Allows you to modify queries so that they return information depending
+  on multiple categories.
+  1. `MIN,MAX`
+  They return the smallest or largest value of a query
+- Look for places from where data might be used to construct a query. Like a textbox, or the url of the website.
```

Comment:
Can we add small examples here and improve clarity?

Reply:
Yep done